napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -128,78 +131,53 @@ def __init__(self, layer: Layer) -> None:
         self.button_grid.setContentsMargins(5, 0, 0, 5)
         self.button_grid.setColumnStretch(0, 1)
         self.button_grid.setSpacing(4)
```

Comment:
```suggestion
        self.button_grid.setSpacing(6)
```
A bit of extra breathing room goes a long way.

| With the change here to 6: | With the previous, 4: |
|----------------------------|----------------------|
| <img width="286" alt="image" src="https://github.com/user-attachments/assets/b7c60d5d-6feb-4445-9bc2-18de78704d51" /> | <img width="282" alt="image" src="https://github.com/user-attachments/assets/b3c0106c-42ca-4f37-8040-f208e2d51a1d" /> |

Heck, i'd do even a bit more...

Reply:
I can understand not wanting to change things in this PR, so this can be ignored/turned into issue/followup.